plugins/modules/omevv_firmware_repository_profile.py

```diff
@@ -396,6 +425,98 @@ def execute(self):
             self.module.exit_json(msg=CHANGES_NOT_FOUND_MSG, diff={"before": {}, "after": {}}, changed=False)
 
 
+class ResyncFirmwareRepositoryProfile(FirmwareRepositoryProfile):
+
+    def __init__(self, module, rest_obj, rest_ome_obj):
+        self.module = module
+        super().__init__(module, rest_obj)
+        self.rest_obj = rest_obj
+        self.ome_obj = rest_ome_obj
+
+    def check_plugin_availability(self):
+        ump_plugin = 0
+        details = self.ome_obj.invoke_request("GET", "PluginService/Plugins")
```

Comment:
Let's put "PluginService/Plugins" into a variable as it is the API endpoint.

Reply:
done